src/cmap/connection_pool_events.ts

```diff
@@ -90,12 +91,14 @@ export class ConnectionClosedEvent extends ConnectionPoolMonitoringEvent {
   connectionId: number | '<monitor>';
   /** The reason the connection was closed */
   reason: string;
+  serviceId?: ObjectId;
 
   /** @internal */
-  constructor(pool: ConnectionPool, connection: Connection, reason: string) {
+  constructor(pool: ConnectionPool, connection: Connection, reason: string, serviceId?: ObjectId) {
```

Comment:
Similarly to the connection events, we have a `serviceId` mounted on `Connection` so no need to add the additional argument

Reply:
This is now pulled off the `Connection`